packages/server/graphql/private/mutations/upgradeToTeamTier.ts

```diff
@@ -85,12 +85,32 @@ const upgradeToTeamTier: MutationResolvers['upgradeToTeamTier'] = async (
         tier: 'team'
       },
       orgId
-    ),
-    removeTeamsLimitObjects(orgId, dataLoader)
+    )
   ])
   organization.tier = 'team'
 
-  await Promise.all([setUserTierForOrgId(orgId), setTierForOrgUsers(orgId)])
+  // If subscription already exists and has open invoices, try to process them
+  const openInvoices = (await manager.listSubscriptionOpenInvoices(stripeSubscriptionId)).data
```

Comment:
Here we added this to upgrade, but we need to do this on card details update `updateCreditCard`
So for example card is expired, we were unable to charge it and locked the org.
User provided a new card by updating details in org settings, we need to retry invoices

Reply:
Makes sense. Should we still include this in `upgradeToTeamTier`? I can see you added it to `oldUpgradeToTeamTier`?

Reply:
@nickoferrall I think we don't need to include it in `upgradeToTeamTier`. It was added to `oldUpgradeToTeamTier` because a single function was responsible for both update details and upgrade